jms/org.wso2.carbon.transport.jms/src/test/java/org/wso2/carbon/transport/jms/test/util/DistributedTxManagerProvider.java

```diff
@@ -34,8 +36,14 @@
     private TransactionManager transactionManager;
 
     protected DistributedTxManagerProvider() {
+        Properties properties = new Properties();
+        properties.setProperty(JMSTestConstants.ATOMIKOS_BASE_DIRECTORY_PROP, JMSTestConstants.TEST_LOG_DIR);
+
         try {
+            UserTransactionServiceImp service = new UserTransactionServiceImp(properties);
+            service.init();
             UserTransactionManager utm = new UserTransactionManager();
+            utm.setStartupTransactionService(false);
```

Comment:
instead of saying utm. Shall we give a proper name? readable names are better than enigmatic names.

Reply:
fixed